doc/user_guide/usage/output.rst

```diff
@@ -21,6 +21,19 @@ a colorized report to stdout at the same time:
 
   --output-format=json:somefile.json,colorized
 
+Environment Variables
+''''''''''''''''''''''''''''
+The colorization of the output can also be controlled through environment
+variables. The precedence for determining output format is as follows:
+
+1. ``NO_COLOR``
+2. ``FORCE_COLOR``
+3. ``--output-format=...``
+
+Setting ``NO_COLOR`` (to any value) will disable colorized output, while
+``FORCE_COLOR`` (to any value) will enable it, overriding the
```

Comment:
Probably worth being extra explicit here:
```suggestion
Setting ``NO_COLOR`` (to any non-empty value) will disable colorized output,
while ``FORCE_COLOR`` (to any non-empty value) will enable it, overriding the
```

*(should this also link https://force-color.org and https://no-color.org websites?)*

Reply:
> Read more details on the https://force-color.org and https://no-color.org websites.

?

Reply:
@stdedos yeah, unless you want to make the var names clickable.

By the way, Sphinx allows documenting env vars. There's `.. envvar::` directive and `:envvar:` role. It's probably a good idea to declare them this way. It would then expose them to things like intersphinx, for example.